subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/accessMetadataArtifact/groovy/build.gradle

```diff
@@ -22,9 +24,9 @@ task printGuavaMetadata {
             Set<ArtifactResult> mavenPomArtifacts = component.getArtifacts(MavenPomArtifact)
             ArtifactResult guavaPomArtifact = mavenPomArtifacts.find { it.file.name == 'guava-18.0.pom' }
             def xml = new XmlSlurper().parse(guavaPomArtifact.file)
-            println guavaPomArtifact.file
+            // println guavaPomArtifact.file
             println xml.name
-            println xml.description
+            println xml.description.toString().stripIndent().trim()
```

Comment:
❓ I like the fact that the assertions are more precise with the changes in this PR but I'm balanced as this makes the snippet a bit more complicated for the sole purpose of the assertions

Reply:
I agree with @eskatos. The tests shouldn't get in the way of clarity.